docs/index.rst

```diff
@@ -47,6 +47,24 @@ A simple example on how to convert MARCXML to JSON:
     [marc21.do(create_record(data)) for data in split_stream(open('/tmp/data.xml', 'r'))]
 
 
+The best way to check if you are not losing metadata during conversion is to
+perform identity check (from MARC21, through master format, to MARC21):
+
+.. code:: shell
+
+    dojson -l marcxml -d marcxml do marc21 do to_marc21 < input.xml > output.xml
+    # Now to see what fields are missing after the conversion
+    xmllint --format output.xml | diff - input.xml
+
+How to get the list of fields that are missing mappings:
+
+.. code:: shell
+
+    dojson -i input.xml -l marcxml missing marc21
```

Comment:
That's a good point, we actually can't since we have 100% coverage for the existing fields.
We can add a test file with fields that is not MARC21 compliant and see if they were removed.